tests/Cms/Plugins/PluginTest.php

```diff
@@ -79,6 +79,18 @@ public function testExtends()
         $this->assertSame($extends, $plugin->extends());
     }
 
+    /**
+     * @covers ::get
+     * @covers ::register
```

Comment:
It uses `register`, but it shouldn't cover it as its behavior is not tested in this method:

```suggestion
```